apps/e2e/cypress/support/utils/createProjectInformation.js

```diff
@@ -28,8 +28,9 @@ export const projectInformation = () => {
 	let applicantFullName = `${applicantFirstName} ${applicantLastName}`;
 	const postcode = 'BS1 6PN';
 	const postcode2 = 'SW1P 4DF';
-	let applicantFullAddress = `2 Temple Quay, Planning Inspectorate, Bristol, ${postcode}`;
-	let applicantFullAddress2 = `2 Marsham Street, Home Office, London, ${postcode2}`;
+	let applicantFullAddress = `2 Temple Quay,        Planning Inspectorate,        Bristol,        ${postcode}`;
+	let applicantFullAddress1 = `2 Temple Quay, Planning Inspectorate, Bristol, ${postcode}`;
+	let applicantFullAddress2 = `2 Marsham Street,        Home Office,        London,        ${postcode2}`;
```

Comment:
It looks like these are not reassigned.

```suggestion
	const applicantFullAddress = `2 Temple Quay,        Planning Inspectorate,        Bristol,        ${postcode}`;
	const applicantFullAddress1 = `2 Temple Quay, Planning Inspectorate, Bristol, ${postcode}`;
	const applicantFullAddress2 = `2 Marsham Street,        Home Office,        London,        ${postcode2}`;
```

Reply:
You are right but following the previous pattern